app/src/main/shared/styles/fonts.scss

```diff
@@ -1,9 +1,9 @@
 @font-face {
     font-family: header;
-    src: url("../resources/PetitaBold.ttf");
+    src: url("../shared/resources/PetitaBold.ttf");
 }
 
 @font-face {
     font-family: body;
-    src: url("../resources/Cloud-Light.otf");
+    src: url("../shared/resources/Cloud-Light.otf");
```

Comment:
because this file gets imported into each app's `style.scss` file, the resource path needs to be relative to an individual app's directory